### DISCUSSION TITLE
Not print the SQL logs when use avram as a stand along ORM.

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1784] [Q&A] [answered]
How to show the SQL query log as development use Rails with ActiveRecord?  

Following is my config for database.

https://github.com/one-in-ten-thousand/production_line_report/blob/master/src/config/database.cr

avram works, but without SQL log.  

Thanks

**Comment by jwoertink** [comment, marked as answer]
You can set the `DB::Log.level = :info` and also set the Avram `Avram::QueryLog.dexter.configure(:info)`.

**Comment by zw963** [comment]
It works, but have to set to :debug instead of :info, thanks!
